src/frontend/src/components/ReactHookEditableList.tsx

```diff
@@ -8,28 +8,51 @@ interface ReactHookEditableListProps {
   register: UseFormRegister<any>;
   append: UseFieldArrayAppend<any, any>;
   remove: UseFieldArrayRemove;
+  title: string;
 }
 
-const ReactHookEditableList: React.FC<ReactHookEditableListProps> = ({ name, ls, register, append, remove }) => {
+const ReactHookEditableList: React.FC<ReactHookEditableListProps> = ({ name, ls, title, register, append, remove }) => {
   return (
     <>
-      {ls.map((_element, i) => {
-        return (
-          <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
-            <TextField required autoComplete="off" sx={{ width: 9 / 10 }} {...register(`${name}.${i}.detail`)} />
-            <IconButton type="button" onClick={() => remove(i)} sx={{ mx: 1, my: 0 }}>
-              <DeleteIcon />
-            </IconButton>
-          </Grid>
-        );
-      })}
+      <Typography variant="h5">{title}</Typography>
+      <Grid item sx={{ my: 1, display: 'flex', flexDirection: 'row', flexWrap: 'wrap' }}>
```

Comment:
ok so the way grids work are you have a <Grid container> with multiple <Grid item> components inside it. Each <Grid container> has 12 columns basically and you can set how many columns each <Grid item> takes up using <Grid item xs={12}> (this means that this item will take up 12 columns when the screen is xtrasmall size). Another example <Grid item xs={12} md={6}> (will take up 6 columns when the screen is medium size or bigger, and 12 columns if the screen size is xtrasmall)

Reply:
so you should be able to make this Grid a container with a columnSpacing prop to define the space between each grid item